docs/enso-protocol.md

```diff
@@ -0,0 +1,14 @@
+Enso protocol consists of pair server and client. They communicate by exchanging
+messages, following JSON-RPC 2.0.
+
+# Setup
+Follow the guidelines of [Enso repository](https://github.com/luna/enso) to setup the project
+. Once you have all the requirements configured, you are able to run the project manager service
```

Comment:
Please do not move dot in doc to separate line. if it is is > 100 chars, move it with lat word.